.github/workflows/ci.yml

```
@@ -62,6 +62,7 @@ jobs:
       amdgpu_families: ${{ matrix.families.family }}
       test_runs_on: ${{ matrix.families.test-runs-on }}
       artifact_run_id: ${{ inputs.artifact_run_id }}
+      expect_failure: ${{ matrix.families.expect_failure == true }}
```

```suggestion
      expect_failure: ${{ matrix.families.expect_failure }}
```
or can a boolean be unset? Should we rather add a default in that case?

---

I believe this'll work, I'll give it a try and check to make sure

---

Never mind, it doesn't work! It actually ignores the jobs where `expect_failure` is false, so reverted it back